### PR TITLE
Remove workaround for loading MatrixCategoryAsAdditiveClosureOfRingAsCategory

### PR DESCRIPTION
**Pull request by zickgraf**

### CODE REVIEW

**Review by mohamed-barakat — APPROVED** [review]

File: gap/SimplicialCategory.gi [hunk 1]
```diff
@@ -21,23 +21,23 @@ InstallGlobalFunction( SimplicialCategoryTruncatedInDegree,
     elif n = 1 then
         F := FreeCategory( RightQuiver( "Delta(C0,C1)[id:C1->C0,s:C0->C1,t:C0->C1]" ) );
         return F /
-               [ [ F.s * F.id, IdentityMorphism( F.C0 ) ],
```
Comment (mohamed-barakat) [inline gap/SimplicialCategory.gi:24]: Why were these changes necessary and part of this PR?

Comment (zickgraf) [reply gap/SimplicialCategory.gi:24]: `\*` for two morphisms is installed in CategoryConstructor, but CategoryConstructor is not a hard dependency (only a transitively suggested one). Before this PR, FreydCategoriesForCAP was loaded including suggested dependencies, which pulled in CategoryConstructor. With this PR, suggested dependencies are not loaded anymore.
